imageio/plugins/ffmpeg.py

```diff
@@ -471,24 +514,26 @@ def _open(self, fps=10, codec='libx264', bitrate=400000):
             # Initialize parameters
             self._proc = None
             self._size = None
-        
-        def _close(self):
-            # Close subprocess
-            if self._proc is not None:
+            self._cmd = None
+
+        def _close(self, timeout=1.0):
+            if self._proc is None:  # pragma: no cover
+                return  # no process
+            if self._proc.poll() is not None:
+                return  # process already dead
+            if self._proc.stdin:
                 self._proc.stdin.close()
-                self._proc.stderr.close()
-                self._proc.wait()
-                self._proc = None
-        
+            self._proc.wait()
```

Comment:
Ah, maybe it should have been the other way around, not calling wait(), but do the "manual" waiting and eventual killing... I'm feeling a bit uncomfortable with the wait; what if it waits forever because ffmpeg refuses to quit?